index.html

```diff
@@ -161,5 +161,20 @@ <h2 class="about-me-title">About Me</h2>
       </ul>
     </ul>
   </section>
+  <section class="contact-me">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form class="contact-form" action="https://formspree.io/f/xknedrly" method="POST">
+      <label>
+        <input class="user-info" type="text" name="fullname" placeholder="Full name" maxlength="30" required>
+      </label>
+      <label>
+        <input class="user-info" type="email" name="email" placeholder="Email addres"  required>
+      </label>
+      <label>
+        <textarea name="message" maxlength="30" required>Enter text here</textarea>
```

Comment:
- [ ] According to the requirements please make sure _**Message field accepts up to 500 chars.**_ and use 500 instead of 30 as the maxlength attribute's value.
- [ ] Please also use a placeholder attribute to display the 'Enter text here' text as currently your required attribute is not working since you already have the placeholder text in your text area element. It should be like `<textarea name="message" maxlength="500" placeholder="Enter text here" required></textarea>` this will use the required attribute and would not allow an empty message to be sent